### DISCUSSION TITLE
Using an array of ELRs for Born-Ferg

### DISCUSSION THREAD

**Discussion by Obalus2022** [casact/chainladder-python#325] [Q&A]
Hi again another question.  Reviewing how chainladder handles BF.  can I use a array of ELRs, or do I have to just use one?

I tried it this way and it didnt work

![image](https://user-images.githubusercontent.com/110035421/184054312-d3763057-4308-4fc0-96b3-c416f76c57a0.png)

**Comment by kennethshsu** [comment]
I think you would have to modify your `sample_weight` directly, as the `apriori` needs to be one single hyperparameter.

You can do something like this, but it's not elegant:

```python
raa = cl.load_sample("raa")
apriori = raa.latest_diagonal * 0 + 50000
apriori.loc[:, :, "1981",] = apriori.loc[:, :, "1981",] * 0.7
apriori.loc[:, :, "1982",] = apriori.loc[:, :, "1982",] * 0.6
cl.BornhuetterFerguson(apriori=1.0).fit(X=raa, sample_weight=apriori).ultimate_
```

Maybe @jbogaardt knows a better way
